src/js/components/Charts/Pie/Donut.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from "react"
```

Comment:
move this file to `/Charts/Donut.js` since it's independent from the Pie index file